src/web/crate_details.rs

```diff
@@ -243,15 +244,23 @@ impl CrateDetails {
 fn map_to_release(conn: &Connection, crate_id: i32, version: String) -> Release {
     let rows = conn
         .query(
-            "SELECT build_status, yanked
+            "SELECT build_status,
+                    yanked,
+                    is_library
              FROM releases
              WHERE releases.crate_id = $1 and releases.version = $2;",
             &[&crate_id, &version],
         )
         .unwrap();
 
-    let (build_status, yanked) = if !rows.is_empty() {
-        (rows.get(0).get(0), rows.get(0).get(1))
+    let (build_status, yanked, is_library) = if !rows.is_empty() {
+        let row = rows.get(0);
+
```

Comment:
nit: `rows.get` is generic:
```suggestion
    let (build_status, yanked, is_library) = if let Some(row) = rows.get(0) {
```
and then you might want to also use `.map_or()` but that's personal preference.